src/backend/distributed/executor/multi_utility.c

```diff
@@ -181,6 +181,33 @@ multi_ProcessUtility(Node *parsetree,
 		return;
 	}
 
+	if (IsA(parsetree, AlterExtensionStmt) || IsA(parsetree, DropStmt))
+	{
+		/*
+		 * In CitusHasBeenLoaded check below, we compare binary Citus version,
+		 * extension version and available version. If they are different, we
+		 * force user to execute ALTER EXTENSION citus UPDATE. Therefore, if
+		 * user executes ALTER EXTENSION or DROP EXTENSION query we should drop
+		 * to the standart utility before CitusHasBeenLoaded check.
+		 */
+		if (IsA(parsetree, AlterExtensionStmt))
```

Comment:
Instead of the nested `if`s, can you write a `bool` called `isAlterOrDropExtension` and assign to it so we can have a single call to `standard_ProcessUtility`?